integration_test/third_party_apps_data/applications/jetty/metadata.yaml

```diff
@@ -0,0 +1,68 @@
+public_url: "https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/jetty"
+short_name: Jetty
+long_name: Jetty
+description: |-
+  The Jetty integration collects
+  access logs. Access logs are parsed into a JSON payload
+  focused on request details. 
+
+  For more information about Jetty, see
+  [The Jetty Docs](https://www.eclipse.org/jetty/documentation/jetty-11/operations-guide/index.html).
+minimum_supported_agent_version:
+  metrics: 2.9.0
```

Comment:
This shouldn't be set up. Our release job will assign the actual release version when we trigger our ops-agent release.

Reply:
minimum_supported_agent_version for new integration or existing integration with newly added logging/metrics support doesn't need to be manually set.
Our release job will take care of.

Reply:
Got it. Would you like me to make a quick PR to fix it?